Example/ReactiveChat/Library/Extensions/UILabel.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  UILabel.swift
+//  ReactiveDataDisplayManagerExample_iOS
+//
+//  Created by Никита Коробейников on 27.03.2023.
+//
+
+import UIKit
+
+public extension UILabel {
+
+    static var base: UILabel = {
+        let label = UILabel()
+        label.textColor = .systemGray
+        label.numberOfLines = 0
+        label.lineBreakMode = .byWordWrapping
+        return label
+    }()
```

Comment:
```suggestion
    }
```
Иначе получается что base хранит переменную, а не создает новые. Также в UIButton

Reply:
Без `()` не скомпилится жеж)
Читай base как дефолтовый лейбл со свойствами по-умолчанию.
Скопировано со старого проекта.

Reply:
ну и `=` убрать сверху)
<img width="503" alt="image" src="https://github.com/surfstudio/ReactiveDataDisplayManager/assets/14955886/7d169a9a-1cac-4eba-b121-3f9967b4ee86">

Reply:
Просто сейчас в бейз хранится объект и при добавлении в другую вью - он пропадет из первой

```swift
let a = UILabel.base
let b = UILabel.base

a === b // true
```

Reply:
Fixed